apps/web/src/pages/Pools/EditPoolPage/components/ScreeningQuestions.tsx

```diff
@@ -0,0 +1,356 @@
+                            required: intl.formatMessage(
+                              errorMessages.required,
+                            ),
+                            validate: {
+                              wordCount: (value: string) =>
+                                maxWordValidator(value, intl),
+                            },
+                          }}
+                        />
+                        <WordLimit locale="fr" index={index} />
+                      </div>
+                    </div>
+                  </Repeater.Fieldset>
+                ))
+              ) : (
+                <Well>
+                  <p
+                    data-h2-font-weight="base(700)"
+                    data-h2-margin-bottom="base(x.5)"
+                  >
+                    {intl.formatMessage({
+                      defaultMessage: "You have no questions.",
+                      id: "Pw/QMl",
+                      description:
+                        "MEssage that appears when there are no screening messages for a pool",
+                    })}
+                  </p>
+                  <p>
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "Start adding some questions using the following button.",
+                      id: "vDqzWG",
+                      description:
+                        "Instructions on how to add a question when there are none",
+                    })}
+                  </p>
+                </Well>
+              )}
+              {!canAdd && (
+                <Well>
+                  <p
+                    data-h2-font-weight="base(700)"
+                    data-h2-margin-bottom="base(x.5)"
+                  >
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "You have reached the maximum amount (3) of screening questions per poster.",
+                      id: "KhQxO8",
+                      description:
+                        "MEssage displayed when a user adds the maximum number of questions",
+                    })}
+                  </p>
+                  <p>
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "Remember, applicants will submit information on how they meet each skill requirement through the regular application process.",
+                      id: "fNYEBT",
+                      description:
+                        "Disclaimer reminding admins of how the application process works when they reach the maximum screening questions",
+                    })}
+                  </p>
+                </Well>
+              )}
+              {removed.length ? (
+                <Well>
+                  {intl.formatMessage(
+                    {
+                      defaultMessage:
+                        "You have removed {number} item(s). Please, remember to save your changes.",
```

Comment:
Should probably have a similar warning for reordering questions.

Reply:
Okay, so I took a bit of a different approach based on this feedback. I went with a more generic message at appears when any changes have been made to this section to avoid having possible 3 alerts appearing (removed, moved, limit hit) 😅 

- [make unsaved message more generic](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6303/commits/27880649e4757e32e4b1bc3ba4d559d499a61ab8)
- [improve unsaved logic, styling, position](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6303/commits/feff0f9ccf59ffcce7de661b74147147469739d1)

Reply:
Love it!